logos/src/logos/main.py

```
@@ -2820,6 +2823,72 @@ async def get_process_id(data: GetProcessIdRequest):
+            detail="No user linked to this key. Service keys cannot log into the UI."
+        )
+    return {
+        "user_id": user["id"],
+        "username": user["username"],
+        "email": user["email"],
+        "role": user["role"],
+        "teams": user["teams"],
+    }
+
+@app.patch("/users/{user_id}/role", tags=["users"])
+async def patch_user_role(user_id: int, body: UpdateRoleRequest, request: Request):
```

Flake8 (7.3.0)

[error] 'UpdateRoleRequest' may be undefined, or defined from star imports: logos.dbutils.dbrequest

(F405)